inst/extdata/abm.Rmd

```diff
@@ -6,10 +6,10 @@ output:
     vertical_layout: scroll
     favicon: kth-logo.png
 params:
-  unit_code: u1g9umtq
-  is_employee: TRUE
-  use_package_data: FALSE
-  embed_data: TRUE
+  unit_code: KTH
+  is_employee: FALSE
+  use_package_data: TRUE
+  embed_data: FALSE
```

Comment:
I think these are good default params

Reply:
Indeed, changed to Sigbritt for testing and forgot to change back. :)

Reply:
Couldn't these defaults (at least the unit_code one) be taken from the new abm_config(), to try to collect defaults in one place? Apparently, the YAML-block can take R variables, as e.g. (taken from https://stackoverflow.com/questions/31861569/setting-document-title-in-rmarkdown-from-parameters):

---
output: html_document
params: 
    set_title: "My Title!"
title: "`r params$set_title`"
---

Reply:
Couldn't these defaults be placed in the new abm_config() (at least unit_code) to have all the defaults in one place.

For instance, this seems to work, which I found in the YAML-docs (https://cran.r-project.org/web/packages/ymlthis/vignettes/yaml-overview.html):

    params:
      unit_code: !r bibliomatrix:::abm_config()$default_unit

@mskyttner @agnepagne